cvat-ui/src/components/job-item/job-actions-menu.tsx

```diff
@@ -17,60 +17,72 @@ interface Props {
     job: Job;
 }
 
+export enum Actions {
+    TASK = 'task',
+    PROJECT = 'project',
+    BUG_TRACKER = 'bug_tracker',
+    IMPORT_JOB = 'import_job',
+    EXPORT_JOB = 'export_job',
+    VIEW_ANALYTICS = 'view_analytics',
+    DELETE = 'delete',
+}
+
 function JobActionsMenu(props: Props): JSX.Element {
     const { job } = props;
 
     const dispatch = useDispatch();
     const history = useHistory();
 
-    const onDelete = useCallback(() => {
-        Modal.confirm({
-            title: `The job ${job.id} will be deleted`,
-            content: 'All related data (annotations) will be lost. Continue?',
-            className: 'cvat-modal-confirm-delete-job',
-            onOk: () => {
-                dispatch(deleteJobAsync(job));
-            },
-            okButtonProps: {
-                type: 'primary',
-                danger: true,
-            },
-            okText: 'Delete',
-        });
-    }, [job]);
+    const onClickMenuWrapper = useCallback(
```

Comment:
```suggestion
    const onClickMenu = useCallback(
```

Reply:
Yes, I think it makes more sense. There is no `wrapping` in this function, its just a callback

Reply:
Renamed. In tasks this name is already occupied by the input parameter.